packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -4329,6 +4332,42 @@ class InheritedElement extends ProxyElement {
     setDependencies(dependent, null);
   }
 
+  /// Called by [dependent] to stop listening to this [InheritedElement].
```

Comment:
nit: If [dependent] is just referencing to the argument of the method it should be enclosed in ` instead of []. We use [] only if the enclosed name is valid in the surrounding scope (e.g. if it is a property on the class).